javascript/hangman.js

```diff
@@ -2,39 +2,75 @@ class Hangman {
   constructor(words) {
     this.words = words;
     // ... your code goes here
```

Comment:
Recuerda que estamos trabajando con POO entonces hay que declarar las propiedades en esta sección
```suggestion
    this.GameOverimg = new Image();
    this.GameOverimg.src = "../images/gameover.png";
    ```